financial-connections/src/main/java/com/stripe/android/financialconnections/features/manualentry/ManualEntryForm.kt

```diff
@@ -0,0 +1,124 @@
+package com.stripe.android.financialconnections.features.manualentry
+
+import androidx.annotation.StringRes
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.compose.runtime.snapshotFlow
+import com.stripe.android.financialconnections.R
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.SharingStarted
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.filterNotNull
+import kotlinx.coroutines.flow.mapLatest
+import kotlinx.coroutines.flow.stateIn
+
+@OptIn(ExperimentalCoroutinesApi::class)
+internal class ManualEntryForm(
+    scope: CoroutineScope
+) {
+
+    private val started = SharingStarted.WhileSubscribed(5_000)
+
+    var routing: String? by mutableStateOf(null)
```

Comment:
nullability on the fields represent if the field is `Dirty` (user has visited it) or not. 

- we don't want to show error messages if the field is not Dirty
- but the form is still invalid, even if the error message is null.